ush/config.sh.RRFS_CONUS_13km

```diff
@@ -1,8 +1,9 @@
 MACHINE="hera"
+version="v0.3.1"
```

Comment:
What is this version? app?

Reply:
This is version for whole system, including source code, workflow, run structure.
The baseline E is v0.3.0. We will try to use version instead of the baseline to avoid confusion in the future.